packages/cursorless-engine/src/languages/TreeSitterQuery/TreeSitterQuery.ts

```diff
@@ -123,7 +125,7 @@ export class TreeSitterQuery {
 }
 
 function normalizeCaptureName(name: string): string {
-  return name.replace(/\.(start|end)$/, "");
+  return name.replace(/\.(start|end)(\.(startof|endof))?$/, "");
```

Comment:
I don't think this works with `@foo.startOf` (without the `.start`)

Reply:
I think that would only be used in cases in which you want a zero width range for an entire scope. And I can't think of any such cases offhand; are there any?

Reply:
oh, haha, the very next comment gives an example of one. ok. :)

Reply:
done